docs/Scaling compute/Polly CLI.md

```diff
@@ -276,8 +300,7 @@ To execute the job, execute the following command
 
 On executing this command, you will be asked to enter the id of the workspace where the job should be run and the path to the job description JSON file. With this, the job will be submitted to run and Job ID will be created. This Job ID will be needed to check the status and the logs of the submitted job.
 
-**Note:**
-*   You do not need to create a new Workspace for running a job. You can simply list the older Workspaces and run a job in an already created Workspace.
+**Note:**   You do not need to create a new Workspace for running a job. You can simply list the older Workspaces and run a job in an already created Workspace.
```

Comment:
Follow the same format for Note as in the rest of the documentation. The text should be as a bullet point in the next line.